### DISCUSSION TITLE
Where to post data issues?

### DISCUSSION THREAD

**Discussion by ChasBelov** [transitland/transitland#372] [Q&A] [answered]
Is this the place to post data issues? Or do we contact the transit agency?

Placer County reorganized their website and the links in their GTFS routes.txt file are now broken.

For example, on the [TransitLand page for Placer County route 40](https://www.transit.land/routes/r-9qcy-40), the GTFS link points to https://www.placer.ca.gov/departments/works/transit/pct/altacolfax
but the actual current route 40 link is https://www.placer.ca.gov/1872/Route-40---Colfax-Alta

Similarly, the [TransitLand page for Sonora HWY 120](https://www.transit.land/routes/r-9qdx-sonorahwy120) link goes to a Not Found.

**Comment by drewda** [comment, marked as answer]
Hi, please contact the transit agencies directly about these issues, since they are within their own feeds. Good luck.
